packages/redux/readme.md

```diff
@@ -1,4 +1,45 @@
-
 # Hops Redux
 
-T.B.D. (To Be Documented)
+hops-redux extends [hops-react](https://github.com/xing/hops/tree/master/packages/react) by providing a rendering context injecting a [Redux](https://github.com/reactjs/redux) [`Provider`](https://github.com/reactjs/react-redux) and some helpers.
+
+Additionally, hops-redux registers the [Thunk](https://github.com/gaearon/redux-thunk) middleware and supports [Redux DevTools](https://github.com/zalmoxisus/redux-devtools-extension) out of the box.
+
+# Installation
+``` bash
+npm install --save hops-redux react-redux redux redux-thunk
```

Comment:
Don't you have to additionally install the dependencies that you need to use hops-react as well? (react-router, hops-react, react-helmet).

Not sure if it's clear that those two build upon each other.